inspirehep/base/templates/format/record/Inspire_Default_HTML_brief_macros.tpl

```diff
@@ -18,8 +18,7 @@
 #}
 
 {% macro render_doi() %}
-  {% if record.get('dois')| is_list() %}
-    {% set filtered_doi = record.get('dois.value')| remove_duplicates() %}
+    {% set filtered_doi = record.get('dois.value')|tranform_nested_list()|remove_duplicates() %}
```

Comment:
Ok cool! What about the filter for flattening a list? As we will not have this problem due to dojson changes, should I remove it, or should we keep it in case we want one day?